core/tutorials/buggyamb/Docs/linux_installation_instructions.md

```diff
@@ -17,9 +17,9 @@ Please check the release information to find out which .NET Core version is requ
 
 ```dotnet --info```
 
-If you don't have ASP.NET Core 3.1 runtime or SDK on your machine then you can find the installation instructions for different Linux distributions in this page: https://docs.microsoft.com/en-us/dotnet/core/install/linux
+If you don't have ASP.NET Core 3.1 runtime or SDK on your machine then you can find the installation instructions for different Linux distributions in [this page](https://docs.microsoft.com/dotnet/core/install/linux)
```

Comment:
```suggestion
If you don't have ASP.NET Core 3.1 runtime or SDK on your machine then you can find the installation instructions for different Linux distributions on [this page](/dotnet/core/install/linux).
```

Reply:
Actually maybe you do need the fully qualified link since this page probably doesn't get published to docs?